docs/Release.md

```diff
@@ -0,0 +1,35 @@
+# Upstream Release Process
+
+## Tag dependent apbs
+
+- keycloak https://github.com/feedhenry/keycloak-apb
+- 3scale https://github.com/feedhenry/3scale-apb
+- sync https://github.com/feedhenry/fh-sync-server-apb
+- digger (mobile ci cd) https://github.com/feedhenry/aerogear-digger-apb  
+
+
+Run the following command in each of the above repos:
+
+```bash
+TAG=0.0.4 #example version
+git checkout $(TAG)
+make build_and_push TAG=$(TAG)
+
+```
+
+For MCP tag as above and run the following commands
+
+```bash
+make image TAG=$(TAG)
```

Comment:
should we also do some sort of git(hub) tag ? 

I generally think those should go hand in hand: create of a github tag, and with "tooling", enusre there will be a matching tag in the dockerhub repo

Reply:
Yeah that would be ideal. At the moment it is just manual